### DISCUSSION TITLE
change elasticsearch.index-name not working

### DISCUSSION THREAD

**Discussion by geosmart** [JanusGraph/janusgraph#2211] [Q&A] [answered]
I use janusgraph 0.3.0
and store the graph in hbase an elasticsearch,

and it create 3 index in es with prefix janusgraph like this:
```
janusgraph_vertex_index
janusgraph_fulltext_index 
janusgraph_edge_index
```
my question is how can to change the index prefix with a custom string

as  [official doc-elasticsearch](https://docs.janusgraph.org/index-backend/elasticsearch/) says
```
JanusGraph index.[X] and index.[X].elasticsearch options
JanusGraph only uses default values for index-name and health-request-timeout. See Configuration Reference for descriptions of these options and their accepted values.

index.[X].elasticsearch.index-name
index.[X].elasticsearch.health-request-timeout
```
I change it by `index.search.elasticsearch.index-name=custom_str` 
but it just not working!

any help is appreciate.

**Comment by li-boxuan** [comment, marked as answer]
This is a GLOBAL_OFFLINE config which requires

> These options can only be changed for the entire database cluster at once when all instances are shut down
